mmv1/templates/terraform/examples/healthcare_dicom_store_bq_stream.tf.erb

```diff
@@ -0,0 +1,28 @@
+resource "google_healthcare_dicom_store" "default" {
+  provider = google-beta
+  name    = "<%= ctx[:vars]['dicom_store_name'] %>"
+  dataset = google_healthcare_dataset.dataset.id
+
+  notification_config {
+    pubsub_topic = google_pubsub_topic.topic.id
+  }
+
+  labels = {
+    label1 = "labelvalue1"
+  }
+
+  stream_configs = {
+    bigquery_destination = {
+      table_uri = "<%= ctx[:vars]['bq_uri'] %>"
+    }
+  }  
+}
+
+resource "google_pubsub_topic" "topic" {
```

Comment:
I believe the google_pubsub_topic and google_healthcare_dataset resources in this example will also need to specify the beta provider in order to have the tests pass.